extension/encoding/jsonlogencodingextension/README.md

```diff
@@ -11,4 +11,33 @@
 [development]: https://github.com/open-telemetry/opentelemetry-collector#development
 <!-- end autogenerated section -->
 
-The `jsonlog` encoding extension is used to marshal/unmarshal JSON log body, ignoring other log fields.
\ No newline at end of file
+The `jsonlog` encoding extension is used to marshal/unmarshal JSON log body, ignoring other log fields.
+
+## Configuration
+
+| Name                     | Description                                     | Default                                      |
+| ------------------------ | ----------------------------------------------- | -------------------------------------------- |
+| raw_log                  | Ignore all attributes and just get body of log  | true                                         |
```

Comment:
Can we make this an enum instead of a boolean? 
something like “body”, “body_with_inline_attributes”. I can see how we might have more modes down the road.

Reply:
I think this makes sense I will update this